### PR TITLE
cleaned up public extern crate; changed that to extern crate only

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

File: src/test/rustdoc/issue-27104.rs [hunk 1]
```diff
@@ -16,5 +16,5 @@
 // @!has - 'extern crate std'
 // @!has - 'use std::prelude::'
 
-// @has - 'pub extern crate empty'
```
Comment (alexcrichton) [inline src/test/rustdoc/issue-27104.rs]: I believe this is specifically testing `pub extern crate`, so this should probably stay.

File: src/test/auxiliary/privacy_reexport.rs [hunk 2]
```diff
@@ -8,7 +8,7 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-pub extern crate core;
```
Comment (alexcrichton) [inline src/test/auxiliary/privacy_reexport.rs]: This also looks like it's also specifically testing the semantics of `pub extern crate`

**Comment by alexcrichton** [comment]
Some of these may have been required for the bootstrap to succeed, are you sure that this builds?

**Comment by petrochenkov** [comment]
The warnings exist only in `stage0` and will disappear with the next snapshot.
No need to fix them, `pub extern crate` works correctly now and can be used.

**Pull request closed without merging**